File: treatment/src/main/java/com/hartwig/actin/treatment/ctc/InterpretedCohortStatus.java
package com.hartwig.actin.treatment.ctc;

import org.immutables.value.Value;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

@Value.Immutable
@Value.Style(passAnnotations = { NotNull.class, Nullable.class })
public abstract class InterpretedCohortStatus {

    public abstract boolean open();

    public abstract boolean slotsAvailable();
}

File: trial/src/main/kotlin/com/hartwig/actin/trial/ctc/InterpretedCohortStatus.kt
package com.hartwig.actin.trial.ctc

import com.hartwig.actin.trial.CTCDatabaseValidationError
import com.hartwig.actin.trial.CtcDatabaseValidation

data class InterpretedCohortStatus(
    val open: Boolean,
    val slotsAvailable: Boolean
)